business/istio_status.go

```diff
@@ -0,0 +1,116 @@
+package business
+
+import (
+	"github.com/kiali/kiali/models"
+	apps_v1 "k8s.io/api/apps/v1"
+
+	"github.com/kiali/kiali/config"
+	"github.com/kiali/kiali/kubernetes"
+)
+
+// SvcService deals with fetching istio/kubernetes services related content and convert to kiali model
+type IstioStatusService struct {
+	k8s kubernetes.IstioClientInterface
+}
+
+type Status string
```

Comment:
I'd really appreciate if we can unify the style of code.
This is nothing wrong, but at some point we need to try to "align" and unify otherwise code will look different.

To me, a wrapper type Status to not use a string is not justified.

Sorry, but in other parts of the code is mapped directly:

```
type Pod struct {
	Name                string            `json:"name"`
	Labels              map[string]string `json:"labels"`
	CreatedAt           string            `json:"createdAt"`
	CreatedBy           []Reference       `json:"createdBy"`
	Containers          []*ContainerInfo  `json:"containers"`
	IstioContainers     []*ContainerInfo  `json:"istioContainers"`
	IstioInitContainers []*ContainerInfo  `json:"istioInitContainers"`
	Status              string            `json:"status"`
	AppLabel            bool              `json:"appLabel"`
	VersionLabel        bool              `json:"versionLabel"`
	Annotations         map[string]string `json:"annotations"`
}
```

I can understand a helper type when it's a complex type (i.e. map/array to pointers) and there is a clear advantaged in the use.

On situations like these, it just a different style and it diverges the style of the code.
Should other developer refactor all Name string, to Name ComponentName ?

So, I don't enter here if this is right or wrong, but let's try to keep an eye at least in the same package and try to unify criterias.